packages/office-ui-fabric-react/src/components/SelectedItemsList/examples/SelectedPeopleList.Controlled.Example.tsx

```diff
@@ -1,102 +1,69 @@
 import {
   IExtendedPersonaProps,
   SelectedPeopleList,
   ISelectedPeopleItemProps,
   ExtendedSelectedItem,
 } from 'office-ui-fabric-react/lib/SelectedItemsList';
-import { Selection } from 'office-ui-fabric-react/lib/Selection';
 import { Stack, IStackStyles } from 'office-ui-fabric-react/lib/Stack';
 import { people, groupOne, groupTwo } from '@uifabric/example-data';
 
 const primaryButtonStyles: Partial<IButtonStyles> = { root: { display: 'block', marginBottom: 20 } };
 const stackStyles: Partial<IStackStyles> = { root: { maxWidth: '100%' } };
+const onRenderItem = (props: ISelectedPeopleItemProps): JSX.Element => {
+  return <ExtendedSelectedItem {...props} />;
+};
+const onCopyItems = (items: IExtendedPersonaProps[]): string => {
+  return items.map((item: IExtendedPersonaProps) => item.text).join(', ');
+};
 
-export interface ISelectedPeopleListControlledExampleState {
-  currentSelectedItems: IExtendedPersonaProps[];
-  nextPersonIndex: number;
-}
-
-export class SelectedPeopleListControlledExample extends React.Component<
-  {},
-  ISelectedPeopleListControlledExampleState
-> {
-  private _selectionList = React.createRef<SelectedPeopleList>();
-  private _selection: Selection;
-
-  constructor(props: {}) {
-    super(props);
-
-    this.state = {
-      currentSelectedItems: [people[40]],
-      nextPersonIndex: 0,
-    };
-    this._selection = new Selection({ onSelectionChanged: () => this.forceUpdate() });
-  }
-
-  public render(): JSX.Element {
-    return (
-      <div>
-        <PrimaryButton
-          text="Add another item"
-          onClick={this._onAddItemButtonClicked}
-          disabled={this.state.nextPersonIndex >= people.length}
-          styles={primaryButtonStyles}
-        />
-        <Stack horizontal wrap styles={stackStyles}>
-          <SelectedPeopleList
-            key="normal"
-            removeButtonAriaLabel="Remove"
-            selectedItems={this.state.currentSelectedItems}
-            componentRef={this._selectionList}
-            onCopyItems={this._onCopyItems}
-            onExpandGroup={this._onExpandItem}
-            copyMenuItemText="Copy"
-            removeMenuItemText="Remove"
-            selection={this._selection}
-            onRenderItem={this._onRenderItem}
-            onItemDeleted={this._onItemDeleted}
-          />
-        </Stack>
-      </div>
-    );
-  }
-
-  private _onRenderItem = (props: ISelectedPeopleItemProps): JSX.Element => {
-    return <ExtendedSelectedItem {...props} />;
-  };
-
-  private _onAddItemButtonClicked = (): void => {
-    const { nextPersonIndex, currentSelectedItems } = this.state;
-    this.setState({
-      currentSelectedItems: [...currentSelectedItems, people[nextPersonIndex]],
-      nextPersonIndex: nextPersonIndex + 1,
-    });
+export const SelectedPeopleListControlledExample: React.FunctionComponent = () => {
+  const [nextPersonIndex, setNextPersonIndex] = React.useState(0);
```

Comment:
Nitpicky thing but spaces between some of these groups of lines would be nice for readability